### PR TITLE
docs(components): update readme

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: libs/components/README.md [hunk 1]
```diff
@@ -1,52 +1,110 @@
...
-### Installation
-
-run:
+
+# Vivid UI
+
+Essential UI **web components** for building modern web applications, bound to provide a **safe**, **simple** and **intuitive** interface.
+
+<!-- ! TODO add visual - GIF or image reflecting an easy integration of vivid in code and page result -->
+![the Vivid logo](/vivid-logo.svg)
```
Comment (yinonov) [inline libs/components/README.md:7]: @Vonage/vivid-design can we add modern front image (like the following but new)?
vivid-cover-wide

**Review by rachelbt — APPROVED** [review]
LGTM

File: change/@vonage-vivid-0d32de44-a0c5-4b02-9f01-f94dfb7e8180.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```
Comment (YonatanKra) [inline change/@vonage-vivid-0d32de44-a0c5-4b02-9f01-f94dfb7e8180.json:2]: It doesn't affect components API AFAIK, `none` would suffice

**Review by YonatanKra — DISMISSED** [review]
LGTM. Maybe we should add (different PR) a quickstart guide that includes the CDN examples?

File: change/@vonage-vivid-0d32de44-a0c5-4b02-9f01-f94dfb7e8180.json (hunk 2 above)
Comment (yinonov) [reply change/@vonage-vivid-0d32de44-a0c5-4b02-9f01-f94dfb7e8180.json:2]:
```suggestion
  "type": "none",
```

**Pull request merged**